routes/favourites-routes.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const checkAuth = require('../middleware/check-auth');
+
+const router = express.Router();
+
+const FavouritesController = require('../controllers/favourites-controller');
+
+router.post('/', checkAuth, FavouritesController.create);
+router.get('/', checkAuth, FavouritesController.list);
```

Comment:
I don't think you use this - it's in the user router too, and the controller uses the route parameter from there, which isn't in here.